src/main/java/com/dataloom/neuron/audit/AuditEntitySet.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2017. Kryptnostic, Inc (dba Loom)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@thedataloom.com
+ */
+
+package com.dataloom.neuron.audit;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.apache.olingo.commons.api.edm.EdmPrimitiveTypeKind;
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.dataloom.authorization.Principal;
+import com.dataloom.authorization.PrincipalType;
+import com.dataloom.data.DatasourceManager;
+import com.dataloom.edm.EntitySet;
+import com.dataloom.edm.type.EntityType;
+import com.dataloom.edm.type.PropertyType;
+import com.dataloom.neuron.signals.Signal;
+import com.datastax.driver.core.utils.UUIDs;
+import com.google.common.base.Optional;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Maps;
+import com.google.common.collect.SetMultimap;
+import com.google.common.collect.Sets;
+import com.kryptnostic.datastore.services.EdmManager;
+
+/*
+ * temporary solution for initializing the Audit EntitySet until we can figure out a better place for initializing
+ * system-wide dependencies
+ */
+public class AuditEntitySet {
+
+    private static final Logger logger = LoggerFactory.getLogger( AuditEntitySet.class );
+
+    private static final String AUDIT_ENTITY_SET_NAME = "Loom Audit Entity Set";
+    private static final String LOOM_AUDIT_NAMESPACE  = "LOOM_AUDIT";
+
+    private static final FullQualifiedName DETAILS_PT_FQN = new FullQualifiedName( LOOM_AUDIT_NAMESPACE, "DETAILS" );
+    private static final FullQualifiedName TYPE_PT_FQN    = new FullQualifiedName( LOOM_AUDIT_NAMESPACE, "TYPE" );
+    private static final FullQualifiedName AUDIT_ET_FQN   = new FullQualifiedName( LOOM_AUDIT_NAMESPACE, "AUDIT" );
+
+    // TODO: where does this belong?
```

Comment:
There probably should be a constants file somewhere that we can put it in. Right now there's `DatastoreConstants` in `loom-api`. It probably makes sense to move that to conductor-client and put all these info in there.